### ISSUE TITLE
About Command

### ISSUE THREAD

**Issue by BillChirico** [VolvoxCommunity/Volvox.Helios#126]
Display a nice embedded message with information about the bot using a command.

Ideas for Information to Display:
- Modules
- Version
- Community Link
- Website Link
- Guild Member Count
- Guild Roles

**Comment by BillChirico** [comment]
Good base from https://leovoel.github.io/embed-visualizer/

```csharp
var builder = new EmbedBuilder()
    .WithDescription("Powerful, modular, web-based, built by a community for a community.")
    .WithColor(new Color(0xA01B01))
    .WithTimestamp(DateTimeOffset.FromUnixTimeMilliseconds(1540855822580))
    .WithThumbnailUrl("http://volvox.tech/images/volvox.png")
    .WithAuthor(author => {
        author
            .WithName("Volvox.Helios")
            .WithUrl("http://discordapp.com");
    })
    .AddInlineField("Version", "v1.0")
```

**Comment by SolarysDev** [comment]
implemented by #140
